### PR TITLE
Update publications.yml

### PR DESCRIPTION
**Pull request by dennisbrookner**
Update the lauedials citation to point to the paper, rather than the pre-print

### CODE REVIEW

File: _data/publications.yml [hunk 1]
```diff
@@ -24,8 +24,8 @@
   title: "Laue-DIALS: open-source software for polychromatic X-ray diffraction data"
   first_author: "Hewitt"
   authors: "**, Rick A.** and Kevin M. Dalton, Derek Mendez, Harrison K. Wang, Margaret A. Klureza, Dennis E. Brookner, Jack B. Greisman, David McDonagh, Vukica Šrajer, Nicholas K. Sauter, Aaron S. Brewster, Doeke R. Hekstra"
-  url: https://www.biorxiv.org/content/10.1101/2024.07.23.604358v1
-  info: bioRxiv (preprint)
+  url: https://pubs.aip.org/aca/sdy/article/11/5/054701/3315187/Laue-DIALS-Open-source-software-for-polychromatic
```
Comment (kmdalton) [inline _data/publications.yml:27]: this URL should use the DOI, [https://doi.org/10.1063/4.0000265](https://doi.org/10.1063/4.0000265)

**Review by kmdalton — CHANGES_REQUESTED** [review]
We should use DOIs when possible for the URLs. They are at least meant to be more permanent / reliable.

**Review by DHekstra — APPROVED** [review]
This now meets Kevin's request (use DOI) and mine (point to actual paper). I think we're done!

**Comment by DHekstra** [comment]
@kmdalton, I think this is waiting for you.

**Review by kmdalton — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**